readme_collector.py

```diff
@@ -0,0 +1,59 @@
+from coc_collector import *
+
+class Readme_Statuses(Enum):
+    """
+    This class enumerates the various potential states of a repository's
+    code of conduct.
+    """
+    GIT_DET_NONE = 1
+    GIT_DET_YES = 2
+
+class Readme_Collector(OSSDI_Collector):
+    """
+    This class collects data on the state of a given repository's Readme
+    file, and populate the corresponding table in the Bossdi database.
+    """
+    def __init__(self, api_key):
+        super().__init__(api_key)
+
+        exists = self.session.query(self.session.query(Readme_State).filter_by\
+                                  (state='GitHub_det_Yes').exists()).scalar()
+        if not exists:
+            self.session.add(Readme_State(state = 'GitHub_det_Yes'))
+            self.session.add(Readme_State(state = 'GitHub_det_No'))
```

Comment:
let's completely write these out.. actually, the table should have an id, shortName (e.g. GH_DETERMINED_NONE), prettyName (e.g. "GitHub Determined None")